domains/mc.katz.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "bananalolok",
+        "discord": "banana_lol_7678"
+    },
```

You need to add an email field
```suggestion
    "owner": {
        "username": "bananalolok",
        "email": "",
        "discord": "banana_lol_7678"
    },
```